src/ImageSharp/Metadata/Profiles/ICC/TagDataEntries/IccNamedColor2TagDataEntry.cs

```diff
@@ -72,7 +71,7 @@ public IccNamedColor2TagDataEntry(string prefix, string suffix, IccNamedColor[]
     /// <param name="suffix">Suffix</param>
     /// <param name="colors">The named colors</param>
     /// <param name="tagSignature">Tag Signature</param>
-    public IccNamedColor2TagDataEntry(int vendorFlags, string prefix, string suffix, IccNamedColor[] colors, IccProfileTag tagSignature)
+    public IccNamedColor2TagDataEntry(int vendorFlags, string? prefix, string? suffix, IccNamedColor[] colors, IccProfileTag tagSignature)
```

Comment:
What's going on here? When would prefix/suffix be null?

Reply:
Have a look at L:51

Reply:
Ahah Thanks! I didn't see cos of the collapsed regions. All good 👍